test/unit/helpers/bignum.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var bignum = require('../../../helpers/bignum');
+/**
+ * Tests required for helpers/bignum.js.
+ *
+ * Note that this test has no dependancies on any runtime and can be run in issolation.
```

Comment:
`Note that this test has no dependencies on any runtime and can be run in isolation.`

Reply:
I have removed the comments as requested below.